developer/README.md

```diff
@@ -4,6 +4,7 @@
 - [Style Guide](contributing/styleguide.md)
 - [Coding Principles](contributing/coding-principles.md)
 - [Decision Records](decision-records/)
+- [Best Practices](./best_practices.md)
```

Comment:
looks like dash is more used than underscore as word separator, I'd stick to that convention